### PR TITLE
Metadatadecoder as part of probe

### PR DESCRIPTION
**Pull request by safchain**

### CODE REVIEW

**Comment by safchain** [comment]
run skydive-scale-tests

**Comment by safchain** [comment]
run skydive-scale-tests

File: analyzer/probes.go [hunk 1]
```diff
@@ -74,5 +111,15 @@ func NewTopologyProbeBundleFromConfig(g *graph.Graph) (*probe.Bundle, error) {
 		}
 	}
 
+	// plugins
+	plugins, err := plugin.LoadTopologyPlugins()
+	if err != nil {
+		return nil, err
+	}
+
+	for _, p := range plugins {
```
Comment (lebauce) [inline analyzer/probes.go]: This should be done in `RegisterProbes`

**Comment by safchain** [comment]
run skydive-scale-tests

**Comment by safchain** [comment]
run skydive-scale-tests

**Comment by safchain** [comment]
run skydive-scale-tests

**Comment by lebauce** [comment]
run skydive-scale-tests

**Comment by safchain** [comment]
run skydive-scale-tests

**Comment by safchain** [comment]
run skydive-scale-tests

**Comment by safchain** [comment]
run skydive-scale-tests

**Comment by safchain** [comment]
run skydive-scale-tests

**Comment by safchain** [comment]
run skydive-scale-tests

**Comment by safchain** [comment]
run skydive-functional-tests-backend-orientdb

**Comment by safchain** [comment]
run skydive-scale-tests

**Comment by safchain** [comment]
run skydive-functional-tests-backend-orientdb

**Pull request merged**